packages/cli/src/modules/modules.config.ts

```diff
@@ -2,7 +2,7 @@ import { CommaSeperatedStringArray, Config, Env } from '@n8n/config';
 import { UnexpectedError } from 'n8n-workflow';
 
 const moduleNames = ['insights'] as const;
-type ModuleName = (typeof moduleNames)[number];
+export type ModuleName = (typeof moduleNames)[number];
```

Comment:
Perhaps we should move these types to `@n8n/api-types` as well. That way we don't need to duplicate the list of module names.